### PR TITLE
uploading utd.png

### PR DESCRIPTION
**Pull request by brands26**
uploading utd.png

### CODE REVIEW

**Comment by mhuggins** [comment]
Is it this one?  https://airdropunited.com/

**Comment by brands26** [comment]
yes it is, and how to add additional info like total supply etc ?

**Comment by mhuggins** [comment]
If you pull/clone the repo, run `yarn start`, it'll regenerate the `tokenList.json` file in the repo.  You can then just commit & PR that file.

**Comment by brands26** [comment]
ok done, lets check it

File: tokenList.json [hunk 1]
```diff
@@ -356,7 +369,7 @@
   },
   "NEX": {
     "symbol": "NEX",
-    "companyName": "Neon Exchange",
+    "companyName": "NEX Token",
```
Comment (mhuggins) [inline tokenList.json]: @comountainclimber @dvdschwrtz It looks like the generated token doesn't match what you guys manually put in here as part of #34.  Was that value manually massaged as part of that PR?  I imagine that if/when the automated CI builds are turned back on, this value will be overwritten as "NEX Token" going forward.

Comment (dvdschwrtz) [reply tokenList.json]: the generated token name looks alright 👍

**Pull request merged**